lib/Elastica/Client.php

```diff
@@ -88,6 +93,21 @@ public function __construct(array $config = [], $callback = null, LoggerInterfac
     }
 
     /**
+     * Get current version
+     *
+     * @return string
+     */
+    public function getVersion()
+    {
+        if ($this->_version) {
+            return $this->_version;
```

Comment:
Is there a reason you "cache" the version? Do you expect that this call is happening more then once during the lifetime of a Client?